octokit.gemspec

```
@@ -6,6 +6,7 @@ require 'octokit/version'
 Gem::Specification.new do |spec|
   spec.add_development_dependency 'bundler', '>= 1', '< 3'
   spec.add_dependency 'sawyer', '>= 0.5.3', '~> 0.8.0'
+  spec.add_dependency 'faraday', '>= 0.9', '< 2.0'
```

This will pick up v0.16.0, which is the affected version. I think we just want to change the constant.

```suggestion
  spec.add_dependency 'faraday', '>= 0.9'
```

If we want a version constraint, we want it as permissive as possible.